native/acter/src/api/push/notification_item.rs

```diff
@@ -234,11 +234,34 @@ impl NotificationItemInner {
         a.reaction_key()
     }
 
-    pub fn new_date(&self) -> Option<UtcDateTime> {
+    pub fn utc_start(&self) -> Option<UtcDateTime> {
         let NotificationItemInner::Activity(a) = &self else {
             return None;
         };
-        a.new_date()
+        let content = a
+            .date_time_range_content()
+            .expect("utc start is available on date time range content");
```

Comment:
this has the ability to return `none`, so `panic`ing in these cases doesn't sound like a good idea. Let's just `?` here, okay?